docs/pico_sdk.md

```diff
@@ -63,21 +66,28 @@ In order to use the RF24 libraries in your RP2040 based project:
 
    Alternatively you can add the RF24* repositories as [git submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules).
 2. Include their CMakeLists.txt files from the RF24 libraries in your project's top-level
-   CMakeLists.txt file (usually located in the "src" directory). The following snippet
-   assumes that your project's "src" directory is on the same level as the previously
-   mentioned "lib" directory.
+   CMakeLists.txt file (usually located in the `src` directory). The following snippet
+   assumes that your project's `src` directory is on the same level as the previously
+   mentioned `lib` directory.
    ```txt
    include(../lib/RF24/CMakeLists.txt)
    include(../lib/RF24Network/CMakeLists.txt)
    include(../lib/RF24Mesh/CMakeLists.txt)
    ```
 3. In the same CMakeLists.txt file from step 2, add the RF24 libraries into the
-   `target_link_libraries` configuration.
+   `target_link_libraries` configuration:
    ```txt
-   target_include_directories(target_name PRIVATE ${CMAKE_CURRENT_LIST_DIR})
+   target_link_libraries(${CMAKE_PROJECT_NAME}
+   ... Your project's other libraries ...
+    RF24
+    RF24Network
+    RF24Mesh
+   )
+   ```
+   If you are using tinyUSB, this line (or similar) should already exist:
+   ```txt
+   target_include_directories(${CMAKE_PROJECT_NAME} PRIVATE ${CMAKE_CURRENT_LIST_DIR})
```

Comment:
I appreciate the simplicity here, so I'm not against this change. However, can there be more than one project in the CMake files? I assume this change means my answer is "no".

Reply:
Good question. Since https://cmake.org/cmake/help/latest/variable/CMAKE_PROJECT_NAME.html mentions "top-level project", I assume that there could be multiple, nested projects. But I'd say for our examples and for most projects created by users and wanting to use the RF24-libraries, this should be fine. And people creating nested projects will probably be skilled enough in CMake to know what to do?

Reply:
> I assume that there could be multiple, nested projects.

We might need to revisit this when we have another "project" for building the Linux examples. However, I believe that would be more of XOR solution.